src/components/Layout/Header/NavBar/Native/BackupPassphraseModal/BackupPassphraseTest.tsx

```diff
@@ -44,7 +44,7 @@ type TestState = {
 
 export const BackupPassphraseTest = ({ vault }: { vault: Vault | null }) => {
   const translate = useTranslate()
-  const history = useHistory()
+  const { goBack: handleBackClick, ...history } = useHistory()
```

Comment:
Technically `history` is `Omit<History, 'description'>` but keep things more declarative to still name it history